includes/Modules/Analytics/Tag_Interface.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Tag_Interface
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics;
+
+/**
+ * Interface for an Analytics tag.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+interface Tag_Interface {
+	/**
+	 * Sets the current home domain.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $domain Domain name.
+	 */
+	public function set_home_domain( $domain );
+
+	/**
+	 * Sets whether or not to anonymize IP addresses.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param bool $anonymize_ip Anonymize or not.
```

Comment:
Nitpick, but maybe let's use a bit more "human" clear language like "Whether to anonymize IP addresses or not.".

Reply:
Agreed! I'll updated it in the other places then as well https://github.com/google/site-kit-wp/blob/d6ea05554e20353df9e770abd193c98b006cdf79/includes/Modules/Analytics/Web_Tag.php#L59